### DISCUSSION TITLE
notice-render-controller returns Error 500

### DISCUSSION THREAD

**Discussion by Micha2** [OP-TED/eForms-SDK#780] [Q&A] [answered]
Hello, I get the following error, when I send a request to the notice-render-controller. I don't know if the error occurs on my side. At the end of my post is the request body.

![grafik](https://github.com/OP-TED/eForms-SDK/assets/48721980/0a752142-4b11-4b88-bbf0-6f36408b2c12)

`{
"file":"PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48VEVEX0VYUE9SVCB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeG1sbnM6eHNpPSJodHRwOi8vd3d3LnczLm9yZy8yMDAxL1hNTFNjaGVtYS1pbnN0YW5jZSIgeG1sbnM9Imh0dHA6Ly9wdWJsaWNhdGlvbnMuZXVyb3BhLmV1L3Jlc291cmNlL3NjaGVtYS90ZWQvUjIuMC45L3B1YmxpY2F0aW9uIiB4bWxuczpuMjAyMT0iaHR0cDovL3B1YmxpY2F0aW9ucy5ldXJvcGEuZXUvcmVzb3VyY2Uvc2NoZW1hL3RlZC8yMDIxL251dHMiIHhzaTpzY2hlbWFMb2NhdGlvbj0iaHR0cDovL3B1YmxpY2F0aW9ucy5ldXJvcGEuZXUvcmVzb3VyY2Uvc2NoZW1hL3RlZC9SMi4wLjkvcHVibGljYXRpb24gVEVEX0VYUE9SVC54c2QiIFZFUlNJT049IlIyLjAuOS5TMDUuRTAxIiBET0NfSUQ9IjczNTM5My0yMDIzIiBFRElUSU9OPSIyMDIzMjM0Ij48VEVDSE5JQ0FMX1NFQ1RJT04+PFJFQ0VQVElPTl9JRD4yMy03MTY5MjItMDAxPC9SRUNFUFRJT05fSUQ+PERFTEVUSU9OX0RBVEU+MjAyMzEyMjI8L0RFTEVUSU9OX0RBVEU+PEZPUk1fTEdfTElTVD5GUiA8L0ZPUk1fTEdfTElTVD48Q09NTUVOVFM+RnJvbSBDb252ZXJ0b3I8L0NPTU1FTlRTPjwvVEVDSE5JQ0FMX1NFQ1RJT04+PExJTktTX1NFQ1RJT04+PFhNTF9TQ0hFTUFfREVGSU5JVElPTl9MSU5LIHhsaW5rOnR5cGU9InNpbXBsZSIgeGxpbms6aHJlZj0iaHR0cHM6Ly90ZWQuZXVyb3BhLmV1IiB4bGluazp0aXRsZT0iVEVEIFdFQlNJVEUiLz48T0ZGSUNJQUxfRk9STVNfTElOSyB4bGluazp0eXBlPSJzaW1wbGUiIHhsaW5rOmhyZWY9Imh0dHBzOi8vdGVkLmV1cm9wYS5ldSIvPjxGT1JNU19MQUJFTFNfTElOSyB4bGluazp0eXBlPSJzaW1wbGUiIHhsaW5rOmhyZWY9Imh0dHBzOi8vdGVkLmV1cm9wYS5ldSIvPjxPUklHSU5BTF9DUFZfTElOSyB4bGluazp0eXBlPSJzaW1wbGUiIHhsaW5rOmhyZWY9Imh0dHBzOi8vdGVkLmV1cm9wYS5ldSIvPjxPUklHSU5BTF9OVVRTX0xJTksgeGxpbms6dHlwZT0ic2ltcGxlIiB4bGluazpocmVmPSJodHRwczovL3RlZC5ldXJvcGEuZXUiLz48L0xJTktTX1NFQ1RJT04+PENPREVEX0RBVEFfU0VDVElPTj48UkVGX09KUz48Q09MTF9PSj5TPC9DT0xMX09KPjxOT19PSj4yMzQ8L05PX09KPjxEQVRFX1BVQj4yMDIzMTIwNTwvREFURV9QVUI+PC9SRUZfT0pTPjxOT1RJQ0VfREFUQT48Tk9fRE9DX09KUz4yMDIzL1MgMjM0LTczNTM5MzwvTk9fRE9DX09KUz48VVJJX0xJU1Q+PFVSSV9ET0MgTEc9IkZSIj5odHRwczovL3RlZC5ldXJvcGEuZXUvdWRsP3VyaT1URUQ6Tk9USUNFOjczNTM5My0yMDIzOlRFWFQ6RlI6SFRNTDwvVVJJX0RPQz48L1VSSV9MSVNUPjxMR19PUklHPkZSPC9MR19PUklHPjxJU09fQ09VTlRSWSBWQUxVRT0iRlIiLz48SUFfVVJMX0dFTkVSQUw+aHR0cDovL2dlcmVkaXMuZnI8L0lBX1VSTF9HRU5FUkFMPjxJQV9VUkxfRVRFTkRFUklORz5odHRwczovL2RlbWF0LWFtcGEuZnI8L0lBX1VSTF9FVEVOREVSSU5HPjxPUklHSU5BTF9DUFYgQ09ERT0iNzEzNTE1MDAiPkdyb3VuZCBpbnZlc3RpZ2F0aW9uIHNlcnZpY2VzPC9PUklHSU5BTF9DUFY+PG4yMDIxOlBFUkZPUk1BTkNFX05VVFMgQ09ERT0iRlIiPkZyYW5jZTwvbjIwMjE6UEVSRk9STUFOQ0VfTlVUUz48bjIwMjE6Q0FfQ0VfTlVUUyBDT0RFPSJGUiI+RnJhbmNlPC9uMjAyMTpDQV9DRV9OVVRTPjwvTk9USUNFX0RBVEE+PENPRElGX0RBVEE+PERTX0RBVEVfRElTUEFUQ0g+MjAyMzExMzA8L0RTX0RBVEVfRElTUEFUQ0g+PERUX0RBVEVfRk9SX1NVQk1JU1NJT04+MjAyMzEyMjIgMTI6MDA8L0RUX0RBVEVfRk9SX1NVQk1JU1NJT04+PEFBX0FVVEhPUklUWV9UWVBFIENPREU9IjgiPk90aGVyPC9BQV9BVVRIT1JJVFlfVFlQRT48VERfRE9DVU1FTlRfVFlQRSBDT0RFPSIzIj5Db250cmFjdCBub3RpY2U8L1REX0RPQ1VNRU5UX1RZUEU+PE5DX0NPTlRSQUNUX05BVFVSRSBDT0RFPSI0Ij5TZXJ2aWNlczwvTkNfQ09OVFJBQ1RfTkFUVVJFPjxQUl9QUk9DIENPREU9IjEiPk9wZW4gcHJvY2VkdXJlPC9QUl9QUk9DPjxSUF9SRUdVTEFUSU9OIENPREU9IjUiPkV1cm9wZWFuIFVuaW9uLCB3aXRoIHBhcnRpY2lwYXRpb24gYnkgR1BBIGNvdW50cmllczwvUlBfUkVHVUxBVElPTj48VFlfVFlQRV9CSUQgQ09ERT0iMSI+U3VibWlzc2lvbiBmb3IgYWxsIGxvdHM8L1RZX1RZUEVfQklEPjxBQ19BV0FSRF9DUklUIENPREU9IjIiPlRoZSBtb3N0IGVjb25vbWljIHRlbmRlcjwvQUNfQVdBUkRfQ1JJVD48TUFfTUFJTl9BQ1RJVklUSUVTIENPREU9IjgiPk90aGVyPC9NQV9NQUlOX0FDVElWSVRJRVM+PEhFQURJTkc+MDFDMDI8L0hFQURJTkc+PElOSVRJQVRPUj4wMTwvSU5JVElBVE9SPjwvQ09ESUZfREFUQT48L0NPREVEX0RBVEFfU0VDVElPTj48VFJBTlNMQVRJT05fU0VDVElPTj48TUxfVElUTEVTPjxNTF9USV9ET0MgTEc9IkJHIj48VElfQ1k+0KRwYdC90YbQuNGPPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD7Qo9GB0LvRg9Cz0Lgg0L/QviDQv9GA0L7Rg9GH0LLQsNC90LUg0L3QsCDQv9C+0YfQstCw0YLQsDwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iQ1MiPjxUSV9DWT5GcmFuY2llPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5Qb3plbW7DrSBwcsWvemt1bTwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iREEiPjxUSV9DWT5GcmFua3JpZzwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+VW5kZXJzw7hnZWxzZXIgYWYgdGVycsOmbjwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iREUiPjxUSV9DWT5GcmFua3JlaWNoPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5Cb2RlbnVudGVyc3VjaHVuZ2VuPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJFTCI+PFRJX0NZPs6TzrHOu867zq/OsTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+zqXPgM63z4HOtc+Dzq/Otc+CIM61zrTOsc+Gzr/Ou86/zrPOuc66z47OvSDOtc+BzrXPhc69z47OvTwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iRU4iPjxUSV9DWT5GcmFuY2U8L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPkdyb3VuZCBpbnZlc3RpZ2F0aW9uIHNlcnZpY2VzPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJFUyI+PFRJX0NZPkZyYW5jaWE8L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPlNlcnZpY2lvcyBkZSBlc3R1ZGlvIGRlIHN1ZWxvczwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iRVQiPjxUSV9DWT5QcmFudHN1c21hYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+TWFhcGlubmEgdXVyaW5ndXRlZW51c2VkPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJGSSI+PFRJX0NZPlJhbnNrYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+TWFhcGVyw6RuIHR1dGtpbXVzcGFsdmVsdXQ8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9IkZSIj48VElfQ1k+RnJhbmNlPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5TZXJ2aWNlcyBkJ8OpdHVkZSBkdSBzb2w8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9IkdBIj48VElfQ1k+RmhyYWluYywgYW48L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPkdyb3VuZCBpbnZlc3RpZ2F0aW9uIHNlcnZpY2VzPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJIUiI+PFRJX0NZPkZyYW5jdXNrYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+VXNsdWdlIGlzdHJhxb5pdmFuamEgdGxhPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJIVSI+PFRJX0NZPkZyYW5jaWFvcnN6w6FnPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5UYWxhai1lbGxlbsWRcnrDqXNpIHN6b2xnw6FsdGF0w6Fzb2s8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9IklUIj48VElfQ1k+RnJhbmNpYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+U2Vydml6aSBkaSBpbnZlc3RpZ2F6aW9uZSBkZWwgdGVycmVubzwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iTFQiPjxUSV9DWT5QcmFuY8WremlqYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+R3J1bnRvIHR5cmltbyBwYXNsYXVnb3M8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9IkxWIj48VElfQ1k+RnJhbmNpamE8L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPkdydW50cyBpenDEk3RlcyBwYWthbHBvanVtaTwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iTVQiPjxUSV9DWT5GcmFuemE8L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPlNlcnZpenppIHRhJyBsLWludmVzdGlnYXp6am9uaSB0YScgbC1hcnQ8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9Ik5MIj48VElfQ1k+RnJhbmtyaWprPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5Cb2RlbW9uZGVyem9la3NkaWVuc3RlbjwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iUEwiPjxUSV9DWT5GcmFuY2phPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5Vc8WCdWdpIGJhZGFuaWEgZ2xlYnk8L1A+PC9USV9URVhUPjwvTUxfVElfRE9DPjxNTF9USV9ET0MgTEc9IlBUIj48VElfQ1k+RnJhbsOnYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+U2VydmnDp29zIGRlIGVzdHVkbyBkZSB0ZXJyZW5vczwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iUk8iPjxUSV9DWT5GcmFuxaNhPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5TZXJ2aWNpaSBkZSBjZXJjZXRhcmUgYSBzb2x1bHVpPC9QPjwvVElfVEVYVD48L01MX1RJX0RPQz48TUxfVElfRE9DIExHPSJTSyI+PFRJX0NZPkZyYW5jw7p6c2tvPC9USV9DWT48VElfVE9XTj5OaW9ydDwvVElfVE9XTj48VElfVEVYVD48UD5Qb3plbW7DvSBwcmllc2t1bTwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iU0wiPjxUSV9DWT5GcmFuY2lqYTwvVElfQ1k+PFRJX1RPV04+TmlvcnQ8L1RJX1RPV04+PFRJX1RFWFQ+PFA+U3Rvcml0dmUgcHJlaXNrYXZlIHRhbDwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PE1MX1RJX0RPQyBMRz0iU1YiPjxUSV9DWT5GcmFua3Jpa2U8L1RJX0NZPjxUSV9UT1dOPk5pb3J0PC9USV9UT1dOPjxUSV9URVhUPjxQPk1hcmt1bmRlcnPDtmtuaW5nc3Rqw6Ruc3RlcjwvUD48L1RJX1RFWFQ+PC9NTF9USV9ET0M+PC9NTF9USVRMRVM+PE1MX0FBX05BTUVTPjxBQV9OQU1FIExHPSJGUiI+R0VSRURJUyBEZXV4LVPDqHZyZXM8L0FBX05BTUU+PC9NTF9BQV9OQU1FUz48L1RSQU5TTEFUSU9OX1NFQ1RJT04+PEZPUk1fU0VDVElPTj48RjAyXzIwMTQgQ0FURUdPUlk9Ik9SSUdJTkFMIiBGT1JNPSJGMDIiIExHPSJGUiI+PExFR0FMX0JBU0lTIFZBTFVFPSIzMjAxNEwwMDI0Ii8+PENPTlRSQUNUSU5HX0JPRFk+PEFERFJFU1NfQ09OVFJBQ1RJTkdfQk9EWT48T0ZGSUNJQUxOQU1FPkdFUkVESVMgRGV1eC1Tw6h2cmVzPC9PRkZJQ0lBTE5BTUU+PEFERFJFU1M+MTcgcnVlIGRlcyBoZXJiaWxsYXV4PC9BRERSRVNTPjxUT1dOPk5JT1JUPC9UT1dOPjxDT1VOVFJZIFZBTFVFPSJGUiIvPjxFX01BSUw+bXRyYW5AZ2VyZWRpcy5mcjwvRV9NQUlMPjxuMjAyMTpOVVRTIENPREU9IkZSIi8+PFVSTF9HRU5FUkFMPmh0dHA6Ly9nZXJlZGlzLmZyPC9VUkxfR0VORVJBTD48L0FERFJFU1NfQ09OVFJBQ1RJTkdfQk9EWT48RE9DVU1FTlRfRlVMTC8+PFVSTF9ET0NVTUVOVD5odHRwczovL2RlbWF0LWFtcGEuZnI8L1VSTF9ET0NVTUVOVD48QUREUkVTU19GVVJUSEVSX0lORk9fSURFTS8+PFVSTF9QQVJUSUNJUEFUSU9OPmh0dHBzOi8vZGVtYXQtYW1wYS5mcjwvVVJMX1BBUlRJQ0lQQVRJT04+PENBX1RZUEVfT1RIRVI+U0FTPC9DQV9UWVBFX09USEVSPjxDQV9BQ1RJVklUWV9PVEhFUj5Sw6lzZWF1eCDDqWxlY3RyaXF1ZXM8L0NBX0FDVElWSVRZX09USEVSPjwvQ09OVFJBQ1RJTkdfQk9EWT48T0JKRUNUX0NPTlRSQUNUPjxUSVRMRT48UD5QcmVzdGF0aW9uIGRlIHLDqWFsaXNhdGlvbiBkJ8OpdHVkZXMgZGUgc29sczwvUD48L1RJVExFPjxSRUZFUkVOQ0VfTlVNQkVSPjIwMjMtMTg8L1JFRkVSRU5DRV9OVU1CRVI+PENQVl9NQUlOPjxDUFZfQ09ERSBDT0RFPSI3MTM1MTUwMCIvPjwvQ1BWX01BSU4+PFRZUEVfQ09OVFJBQ1QgQ1RZUEU9IlNFUlZJQ0VTIi8+PFNIT1JUX0RFU0NSPjxQPlByZXN0YXRpb24gZGUgcsOpYWxpc2F0aW9uIGRlIG1pc3Npb25zIGQnw6l0dWRlcyBkZSBzb2xzPC9QPjwvU0hPUlRfREVTQ1I+PExPVF9ESVZJU0lPTj48TE9UX0FMTC8+PC9MT1RfRElWSVNJT04+PE9CSkVDVF9ERVNDUiBJVEVNPSIxIj48VElUTEU+PFA+w6l0dWRlcyBkZSBzb2wgZ8OpbsOpcmFsZXMgPC9QPjwvVElUTEU+PExPVF9OTz4xPC9MT1RfTk8+PENQVl9BRERJVElPTkFMPjxDUFZfQ09ERSBDT0RFPSI3MTM1MTUwMCIvPjwvQ1BWX0FERElUSU9OQUw+PG4yMDIxOk5VVFMgQ09ERT0iRlIiLz48TUFJTl9TSVRFPjxQPkTDqXBhcnRlbWVudCBkZXMgRGV1eC1Tw6h2cmVzLjwvUD48L01BSU5fU0lURT48U0hPUlRfREVTQ1I+PFA+UHJlc3RhdGlvbiBkZSByw6lhbGlzYXRpb24gZOKAmcOpdHVkZXMgZGUgc29sczwvUD48L1NIT1JUX0RFU0NSPjxBQz48QUNfUFJPQ1VSRU1FTlRfRE9DLz48L0FDPjxEVVJBVElPTiBUWVBFPSJNT05USCI+MTI8L0RVUkFUSU9OPjxSRU5FV0FMLz48UkVORVdBTF9ERVNDUj48UD5MZSBjb250cmF0IGVzdCBjb25jbHUgcG91ciB1bmUgZHVyw6llIGRlIDEyIG1vaXMgw6AgY29tcHRlciBkZSBsYSBub3RpZmljYXRpb24gZHUgY29udHJhdC48L1A+PFA+TGVzIENvbnRyYXRzIHNlcm9udCByZWNvbmR1Y3RpYmxlcyB0YWNpdGVtZW50IHRyb2lzIGZvaXMgcG91ciB1bmUgcMOpcmlvZGUgZOKAmXVuIGFuICgyMDI1LCAyMDI2IGV0IDIwMjcpLCBzYXVmIGTDqW5vbmNpYXRpb24gZXhwcmVzc2UgZGUgR0VSRURJUyBwYXIgbGV0dHJlIHJlY29tbWFuZMOpZSwgYXZlYyB1biBwcsOpYXZpcyBk4oCZdW4gbW9pcyBhdmFudCBsYSBmaW4gZGUgbGEgcMOpcmlvZGUgaW5pdGlhbGUgb3UgZGUgbGEgcMOpcmlvZGUgZGUgcmVjb25kdWN0aW9uLiA8L1A+PC9SRU5FV0FMX0RFU0NSPjxOT19BQ0NFUFRFRF9WQVJJQU5UUy8+PE5PX09QVElPTlMvPjxOT19FVV9QUk9HUl9SRUxBVEVELz48L09CSkVDVF9ERVNDUj48T0JKRUNUX0RFU0NSIElURU09IjIiPjxUSVRMRT48UD7DqXR1ZGVzIHNww6ljaWZpcXVlIHRyYXZlcnPDqWVzIGZlcnJvdmlhaXJlczwvUD48L1RJVExFPjxMT1RfTk8+MjwvTE9UX05PPjxDUFZfQURESVRJT05BTD48Q1BWX0NPREUgQ09ERT0iNzEzNTE1MDAiLz48L0NQVl9BRERJVElPTkFMPjxuMjAyMTpOVVRTIENPREU9IkZSIi8+PE1BSU5fU0lURT48UD5Ew6lwYXJ0ZW1lbnQgZGVzIERldXgtU8OodnJlcy48L1A+PC9NQUlOX1NJVEU+PFNIT1JUX0RFU0NSPjxQPlByZXN0YXRpb24gZGUgcsOpYWxpc2F0aW9uIGTigJnDqXR1ZGVzIGRlIHNvbHM8L1A+PC9TSE9SVF9ERVNDUj48QUM+PEFDX1BST0NVUkVNRU5UX0RPQy8+PC9BQz48RFVSQVRJT04gVFlQRT0iTU9OVEgiPjEyPC9EVVJBVElPTj48UkVORVdBTC8+PFJFTkVXQUxfREVTQ1I+PFA+TGUgY29udHJhdCBlc3QgY29uY2x1IHBvdXIgdW5lIGR1csOpZSBkZSAxMiBtb2lzIMOgIGNvbXB0ZXIgZGUgbGEgbm90aWZpY2F0aW9uIGR1IGNvbnRyYXQuPC9QPjxQPkxlcyBDb250cmF0cyBzZXJvbnQgcmVjb25kdWN0aWJsZXMgdGFjaXRlbWVudCB0cm9pcyBmb2lzIHBvdXIgdW5lIHDDqXJpb2RlIGTigJl1biBhbiAoMjAyNSwgMjAyNiBldCAyMDI3KSwgc2F1ZiBkw6lub25jaWF0aW9uIGV4cHJlc3NlIGRlIEdFUkVESVMgcGFyIGxldHRyZSByZWNvbW1hbmTDqWUsIGF2ZWMgdW4gcHLDqWF2aXMgZOKAmXVuIG1vaXMgYXZhbnQgbGEgZmluIGRlIGxhIHDDqXJpb2RlIGluaXRpYWxlIG91IGRlIGxhIHDDqXJpb2RlIGRlIHJlY29uZHVjdGlvbi4gPC9QPjwvUkVORVdBTF9ERVNDUj48Tk9fQUNDRVBURURfVkFSSUFOVFMvPjxOT19PUFRJT05TLz48Tk9fRVVfUFJPR1JfUkVMQVRFRC8+PC9PQkpFQ1RfREVTQ1I+PC9PQkpFQ1RfQ09OVFJBQ1Q+PExFRlRJPjxFQ09OT01JQ19DUklURVJJQV9ET0MvPjxURUNITklDQUxfQ1JJVEVSSUFfRE9DLz48L0xFRlRJPjxQUk9DRURVUkU+PFBUX09QRU4vPjxDT05UUkFDVF9DT1ZFUkVEX0dQQS8+PERBVEVfUkVDRUlQVF9URU5ERVJTPjIwMjMtMTItMjI8L0RBVEVfUkVDRUlQVF9URU5ERVJTPjxUSU1FX1JFQ0VJUFRfVEVOREVSUz4xMjowMDwvVElNRV9SRUNFSVBUX1RFTkRFUlM+PExBTkdVQUdFUz48TEFOR1VBR0UgVkFMVUU9IkZSIi8+PC9MQU5HVUFHRVM+PERVUkFUSU9OX1RFTkRFUl9WQUxJRCBUWVBFPSJNT05USCI+NjwvRFVSQVRJT05fVEVOREVSX1ZBTElEPjxPUEVOSU5HX0NPTkRJVElPTj48REFURV9PUEVOSU5HX1RFTkRFUlM+MjAyMy0xMi0yMjwvREFURV9PUEVOSU5HX1RFTkRFUlM+PFRJTUVfT1BFTklOR19URU5ERVJTPjE0OjAwPC9USU1FX09QRU5JTkdfVEVOREVSUz48L09QRU5JTkdfQ09ORElUSU9OPjwvUFJPQ0VEVVJFPjxDT01QTEVNRU5UQVJZX0lORk8+PFJFQ1VSUkVOVF9QUk9DVVJFTUVOVC8+PEVTVElNQVRFRF9USU1JTkc+PFA+TGVzIENvbnRyYXRzIHNlcm9udCByZWNvbmR1Y3RpYmxlcyB0YWNpdGVtZW50IHRyb2lzIGZvaXMgcG91ciB1bmUgcMOpcmlvZGUgZOKAmXVuIGFuICgyMDI1LCAyMDI2IGV0IDIwMjcpLCBzYXVmIGTDqW5vbmNpYXRpb24gZXhwcmVzc2UgZGUgR0VSRURJUyBwYXIgbGV0dHJlIHJlY29tbWFuZMOpZSwgYXZlYyB1biBwcsOpYXZpcyBk4oCZdW4gbW9pcyBhdmFudCBsYSBmaW4gZGUgbGEgcMOpcmlvZGUgaW5pdGlhbGUgb3UgZGUgbGEgcMOpcmlvZGUgZGUgcmVjb25kdWN0aW9uLiA8L1A+PC9FU1RJTUFURURfVElNSU5HPjxBRERSRVNTX1JFVklFV19CT0RZPjxPRkZJQ0lBTE5BTUU+VHJpYnVuYWwgQWRtaW5pc3RyYXRpZjwvT0ZGSUNJQUxOQU1FPjxBRERSRVNTPjE1LCBSdWUgZGUgQmxvc3NhYzwvQUREUkVTUz48VE9XTj5QT0lUSUVSUzwvVE9XTj48UE9TVEFMX0NPREU+ODYwMDA8L1BPU1RBTF9DT0RFPjxDT1VOVFJZIFZBTFVFPSJGUiIvPjwvQUREUkVTU19SRVZJRVdfQk9EWT48UkVWSUVXX1BST0NFRFVSRT48UD5QcsOpY2lzaW9ucyBjb25jZXJuYW50IGxlcyBkw6lsYWlzIGQnaW50cm9kdWN0aW9uIGRlIHJlY291cnM6PC9QPjxQPkxlIG1hcmNow6kgcG91cnJhIGZhaXJlIGzigJlvYmpldCA6PC9QPjxQPi0gZCd1biByw6lmw6lyw6kgcHLDqWNvbnRyYWN0dWVsLCBqdXNxdSfDoCBsYSBzaWduYXR1cmUgZHUgbWFyY2jDqSBjb25mb3Jtw6ltZW50IGF1eCBkaXNwb3NpdGlvbnMgZGVzIGFydGljbGVzIEwuNTUxLTEgZXQgUi41NTEtMSBldCBzdWl2YW50cyBkdSBDb2RlIGRlIEp1c3RpY2UgQWRtaW5pc3RyYXRpdmUgOzwvUD48UD4tIGQndW4gcsOpZsOpcsOpIGNvbnRyYWN0dWVsIMOgIGNvbXB0ZXIgZGUgbGEgc2lnbmF0dXJlIGR1IG1hcmNow6kgY29uZm9ybcOpbWVudCBhdXggYXJ0aWNsZXMgTC41NTEtMTMgZXQgUi41NTEtNyBldCBzdWl2YW50cyBkdSBDb2RlIGRlIEp1c3RpY2UgQWRtaW5pc3RyYXRpdmUgZGFucyB1biBkw6lsYWkgZGUgdHJlbnRlIGV0IHVuIGpvdXJzIMOgIGNvbXB0ZXIgZGUgbGEgcHVibGljYXRpb24gZGUgbCdhdmlzIGQnYXR0cmlidXRpb24gZHUgbWFyY2jDqS48L1A+PFA+LSBk4oCZdW4gcmVjb3VycyBkZSBwbGVpbmUganVyaWRpY3Rpb24gZW4gY29udGVzdGF0aW9uIGRlIGxhIHZhbGlkaXTDqSBkdSBjb250cmF0IGRhbnMgdW4gZMOpbGFpIGRlIDIgbW9pcyDDoCBjb21wdGVyIGRlcyBtZXN1cmVzIGRlIHB1YmxpY2l0w6kgYXBwcm9wcmnDqWVzIChDRSwgQXNzLiwgNCBhdnJpbCAyMDE0LCBEw6lwYXJ0ZW1lbnQgZGUgVGFybiBldCBHYXJvbm5lLCBuwrAgMzU4OTk0KS48L1A+PC9SRVZJRVdfUFJPQ0VEVVJFPjxBRERSRVNTX1JFVklFV19JTkZPPjxPRkZJQ0lBTE5BTUU+R0VSRURJUyAtIFDDtGxlIEp1cmlkaXF1ZTwvT0ZGSUNJQUxOQU1FPjxBRERSRVNTPjE3LCBSdWUgZGVzIEhlcmJpbGxhdXg8L0FERFJFU1M+PFRPV04+TklPUlQ8L1RPV04+PFBPU1RBTF9DT0RFPjc5MDAwPC9QT1NUQUxfQ09ERT48Q09VTlRSWSBWQUxVRT0iRlIiLz48L0FERFJFU1NfUkVWSUVXX0lORk8+PERBVEVfRElTUEFUQ0hfTk9USUNFPjIwMjMtMTEtMzA8L0RBVEVfRElTUEFUQ0hfTk9USUNFPjwvQ09NUExFTUVOVEFSWV9JTkZPPjwvRjAyXzIwMTQ+PC9GT1JNX1NFQ1RJT04+PC9URURfRVhQT1JUPg==",
  "language": "en",
  "format": "HTML"
}`

**Comment by YvesJo** [comment, marked as answer]
For this question not immediately related to the SDK, please get directly in touch with the helpdesk ted@publications.europa.eu

**Comment by pcolast** [comment]
Good Day,
The endpoint you tried works only with eForms notice. The file you provided is however not an eForms notice.

Have a nice day,
Stefan

**Reply by Micha2** [reply]
Thanks, that's the answer. I tried it with notices in the eForms format and it works.
